File: C & C++ PRACTICE CODES/INVERTED TRIANGLE PATTERN/code3.c
#include <stdio.h>


void main(){

	int row ;
	printf("Enter no. of rows : ");
	scanf("%d",&row);

	for(int i = 0 ; i < row ; i++){

		for(int j = 0 ; j < row - i ; j++){

			if(i%2 == 0){

				printf("%c\t",65+i);

			}else{

				printf("%c\t",97+i);
			}
		}

		printf("\n\n");
	}

}

File: JAVA DAILYCODES ASSGNPRATICE QUIZE DSA/Basic is else assignments/8/code3.java
import java.util.Scanner;

class NestedDemo{

	public static void main(String[] args){

		Scanner s = new Scanner(System.in);
		System.out.println("enter no. of rows");
		int row = s.nextInt();

		for(int i = 0; i < row ; i++){
			for(int j = row;j >=1 ;j--){

				System.out.print((char)(j+64)+"	");

			}
			System.out.println("\n");
		}
	}
}
